kinbot/conformers.py

```diff
@@ -426,31 +426,41 @@ def check_conformers(self, wait=0):
-                                    if freq[0] >= 0.:
-                                        err = -1
-                                    if self.species.natom > 2 and freq[1] <= 0.:
-                                        err = -1
+                                # job fails if conformer freq array is empty
+                                if len(freq) > 0:
+                                    if self.species.wellorts:
+                                        if freq[0] >= 0.:
+                                            err = -1
+                                        if self.species.natom > 2 and freq[1] <= 0.:
+                                            err = -1
+                                    else:
+                                        if freq[0] <= 0.:
+                                            err = -1
                                 else:
-                                    if freq[0] <= 0.:
-                                        err = -1
+                                    logging.info("Conformer {} failed due to empty freq array".format(ci))
+                                    err = -1
                             if err == 0:
                                 lowest_totenergy = energy + zpe
                                 if self.species.wellorts:
                                     base_imag_freq = freq[0]
                         if energy + zpe < lowest_totenergy:
                             err, freq = self.qc.get_qc_freq(job, self.species.natom)
                             ratio = 0.8
-                            if self.species.wellorts:
-                                if freq[0] / base_imag_freq < ratio:
-                                    err = -1 
-                                if freq[0] / base_imag_freq > 1. / ratio:
-                                    err = -1 
-                                if self.species.natom > 2 and freq[1] <= 0.:
-                                    err = -1
+                            # job fails if conformers freq array is empty
+                            if len(freq) > 0:
+                                if self.species.wellorts:
+                                    if freq[0] / base_imag_freq < ratio:
+                                        err = -1 
+                                    if freq[0] / base_imag_freq > 1. / ratio:
+                                        err = -1 
+                                    if self.species.natom > 2 and freq[1] <= 0.:
+                                        err = -1
+                                else:
+                                    if freq[0] <= 0.:
+                                        err = -1
                             else:
-                                if freq[0] <= 0.:
-                                    err = -1
+                                logging.info("Conformer {} failed due to empty freq array".format(ci))
+                                err = -1
```

Comment:
I was getting an error that for a conformer the length of `freq` was 0, so I added a check for that. 

I'm not completely sure why the freq array was not empty for the calculations where this happened. I'm wondering if it is a similar error to the one we see where a geometry is set to `0.00, 0.00, 0.00` for all atoms. 

This check allowed the code to continue despite the error and just treats the conformer as if the calculation fails/is not the lowest energy structure.